lib/install/is-fs-access-available.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+var fs = require('fs')
+var process = require('process')
+var semver = require('semver')
+
+var isWindows = process.platform === 'win32'
+
+// fs.access first introduced in node 0.12 / io.js
+if (!fs.access) {
+  module.exports = false
+} else if (!isWindows) {
+  // fs.access always works on non-Windows OSes
+  module.exports = true
+} else {
+  // The Windows implementation of `fs.access` has a bug where it will
+  // sometimes return access errors all the time for directories, even
```

Comment:
@lukeapage
If `fs.access` is just blindly returning `true` on Windows, then yes, that is a problem. I'm going to have to go digging through the `libuv` source, clearly.

The only problem with the permissions checks being too permissive, is that you get failures later rather than earlier. It shouldn't otherwise cause issues, but I'd still like to fix it. My fallback code just tries making a file in the directory to see if it has permission, and perhaps we'll have to permanently use that.